packages/vite/package.json

```diff
@@ -31,15 +31,15 @@
   },
   "homepage": "https://github.com/preactjs/prefresh#readme",
   "dependencies": {
-    "@babel/core": "^7.9.6",
+    "@babel/core": "^7.22.1",
     "@prefresh/babel-plugin": "0.5.0",
     "@prefresh/core": "^1.5.1",
     "@prefresh/utils": "^1.2.0",
-    "@rollup/pluginutils": "^4.1.0"
+    "@rollup/pluginutils": "^4.2.1"
   },
   "devDependencies": {
-    "preact": "^10.5.7",
-    "vite": "^2.0.0-beta.3"
+    "preact": "^10.15.1",
+    "vite": "^4.3.9"
```

Comment:
Doing so will force us to bump peerDeps here to be OR over 2 OR over 3 Or over 4

Reply:
fixed
https://github.com/preactjs/prefresh/pull/507/commits/3130ed739c09ad586ebfbfb3895bcbbf4493deab